custom/icds_reports/utils/aggregation_helpers/delivery_forms.py

```diff
@@ -38,7 +38,7 @@ def aggregation_query(self):
             SUM(CASE WHEN (unscheduled_visit=0 AND days_visit_late < 8) OR
                           (timeend::DATE - next_visit) < 8 THEN 1 ELSE 0 END
                 ) OVER w as valid_visits,
-            where_born
+            LAST_VALUE(where_born) OVER w AS where_born
```

Comment:
fixes issue where a woman has multiple delivery forms associated with her ccs_record. defaults to the last submitted entry

Reply:
Hey, how can two delivery form be submitted for one mother in one mother?

Reply:
I'm not sure, but there were multiple occurrences where this was the case